Documentation/Blazorise.Docs/Components/ComponentApiDocsProperty.razor

```diff
@@ -0,0 +1,59 @@
+﻿@using Blazorise.Docs.Models.ApiDocsDtos
+
+@if ( filteredApiDocsForComponents.Any() )
+{
+    <Heading Margin="Margin.Is2.FromTop.Is3.FromBottom" Size="HeadingSize.Is3">
+        @(PropertyType == PropertyTypes.Events
+            ? "Events"
+            : "Parameters")
+    </Heading>
+}
+
+@foreach ( var apiDocsForComponent in filteredApiDocsForComponents )
+{
+    <DocsAttributes ShowDefaults="@(PropertyType != PropertyTypes.Events)"
+                    Name="@(PropertyType == PropertyTypes.Events ? "Event": "Parameter")"
+                    Title="@(MultipleComponentTypes ? apiDocsForComponent.TypeName : null)">
+        @foreach ( var property in ( PropertyType == PropertyTypes.Events ? apiDocsForComponent.Events : apiDocsForComponent.Parameters ).OrderBy( x => x.Name ) )
+        {
+            <DocsAttributesItem @key="@property.Name" Name="@property.Name" Type="@property.TypeName"
+                                Default="@property.DefaultValueString">
+                <Paragraph Margin="@(property.IsBlazoriseEnum ? null : Margin.Is0.FromBottom)">
+                    @((MarkupString)property.Summary)
+                </Paragraph>
+
+                @if (property.IsBlazoriseEnum)
+                {
+                    var enumNames = Enum.GetNames(property.Type);
+
+                    <Paragraph Margin="Margin.Is0.FromBottom">
+                        <Text>Possible values:</Text>
+                        @for ( int enumValue = 0; enumValue < enumNames.Length; enumValue++ )
+                        {
+                            int i1 = enumValue;
```

Comment:
Actually **var i** was fine. as most programmers are acquainted with that for loops.

I meant that the **i1** wasn't clear for someone that would not understand that blazor needs to somehow capture the value "inline". So I meant to clarify the i1 with localIdx, localI, or in this case localEnumValue, or something along those lines, just so it doesn't make it as obscure, and an unsuspecting programmer tries to remove the i1, thinking it's redudant to have it.